gcp/client.py

```diff
@@ -1,12 +1,6 @@
-import warnings
-
 from apiclient.discovery import build as build_service
 from apiclient.errors import HttpError
 
-warnings.filterwarnings(
-    "ignore", "Your application has authenticated using end user credentials"
-)
-
```

Comment:
Question: what's the thinking of moving this to `__init__.py`. I.e. I would have thought this code belonged in `client.py` based on our past discussions.

I'm not against it, I just want to understand it so I can use it properly.  Maybe a doc update is needed?

Reply:
I'm open to either. I did worry that moving it to `__init__.py` might be hiding it.

>I.e. I would have thought this code belonged in `client.py` based on our past discussions.

Would you mind reminding me about the past discussion? I don't remember discussing this piece as it relates to which file it should live in.

Reply:
Roughly (my interpretation):
- `__init__.py` is just to mark directory as a package
- `client.py` has all the code related to communication with the service
- `helpers.py` has all the code related to simplifying tests (e.g. operations on data structures returned from service)

But I see that differs a bit from the [design docs](https://mozilla.github.io/frost/readme-include.html#design).

Again, this isn't a blocker for this commit, but it does look like the design & [file layout](https://mozilla.github.io/frost/readme-include.html#file-layout) needs a refresh.  I'll open a separate issue.

Reply:
>`client.py` has all the code related to communication with the service

Yeah that's a great point. I think it would be best to add these lines back to `client.py`. Will do that now.